sync_ratings.py

```diff
@@ -91,16 +91,21 @@ def setup_logging(self):
 			self.logger.addHandler(ch_std)
 
 	def sync(self):
+		if self.options.sync not in ["both", "tracks","playlists"]:
+			raise RuntimeError('Invalid sync item selected: {}'.format(self.options.sync ))
+			
 		self.local_player.connect()
 		self.remote_player.connect(
 			server=self.options.server,
 			username=self.options.username,
 			password=self.options.passwd,
 		)
-		self.sync_tracks()
+		if not "playlists" in self.options.sync:
+			self.sync_tracks()
```

Comment:
This is semantically easier to read
```suggestion
		if "playlists" not in self.options.sync:
			self.sync_tracks()
```